Carmu/Sources/Views/CrewMake/StopoverPointCheckView.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  StopoverPointCheckView.swift
+//  Carmu
+//
+//  Created by 김동현 on 11/2/23.
+//
+
+import UIKit
+
+import SnapKit
+
+final class StopoverPointCheckView: UIView {
+
+    private lazy var titleStackView = UIStackView()
+
+    private lazy var titleLabel1 = CrewMakeUtil.defalutTitle(titleText: "여정의 ")
+    private lazy var titleLabel2 = CrewMakeUtil.accPrimaryTitle(titleText: "경유지")
+    private lazy var titleLabel3 = CrewMakeUtil.defalutTitle(titleText: "가 있으신가요?")
+
+    private let noStopoverButton = LargeSelectButton(
+        topTitle: "아니오",
+        bottomTitle: "넘어가기",
+        imageName: "NoStopoverPoint"
+    )
+
+    private let yesStopoverButton = LargeSelectButton(
+        topTitle: "네",
+        bottomTitle: "경유지 추가하기",
+        imageName: "YesStopoverPoint"
+    )
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+
+    override func draw(_ rect: CGRect) {
+        setupViews()
+        setAutoLayout()
+    }
+
+    private func setupViews() {
+        titleStackView.axis = .horizontal
+
+        titleStackView.addArrangedSubview(titleLabel1)
+        titleStackView.addArrangedSubview(titleLabel2)
+        titleStackView.addArrangedSubview(titleLabel3)
+
+        addSubview(titleStackView)
+        addSubview(noStopoverButton)
+        addSubview(yesStopoverButton)
+    }
+
+    private func setAutoLayout() {
+
+        titleStackView.snp.makeConstraints { make in
+            make.top.equalTo(safeAreaLayoutGuide).inset(78)
+            make.leading.equalToSuperview().inset(20)
+        }
+
+        noStopoverButton.snp.makeConstraints { make in
+            make.leading.equalToSuperview().inset(20)
+            make.top.equalTo(titleStackView.snp.bottom).offset(176)
+            make.width.equalTo(165)
+            make.height.equalTo(240)
+        }
+
+        yesStopoverButton.snp.makeConstraints { make in
+            make.trailing.equalToSuperview().inset(20)
+            make.top.equalTo(noStopoverButton.snp.top)
```

Comment:
```suggestion
            make.top.equalTo(noStopoverButton)
```
`yesStopoverButton`의 top에 대한 조건이고, equalTo 안에 들어가는 것도 `noStopoverButton`의 top을 의미한다면 SnapKit에서 암시 기능을 지원하기 때문에 위처럼 생략해줘도 될 것 같습니다🙃